packages/flutter/lib/src/widgets/expansible.dart

```diff
@@ -0,0 +1,417 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// @docImport 'package:flutter/material.dart';
+library;
+
+import 'basic.dart';
+import 'framework.dart';
+import 'gesture_detector.dart';
+import 'page_storage.dart';
+import 'ticker_provider.dart';
+import 'transitions.dart';
+
+/// The type of the callback that returns the header or body of an [Expansible].
+///
+/// The `isExpanded` property allows for a different header or body depending on
+/// whether the [Expansible] is currently expanded or collapsed.
+///
+/// The `animation` property exposes the underlying animation controller, which
```

Comment:
Should be more specific
```
The `animation` property exposes the underlying expanding or collapsing animation where 0 is completely collapsed and 1 is completely expanded. This can be used ....
```